rotkehlchen/csv_exporter.py

```diff
@@ -94,14 +95,15 @@ def _dict_to_csv_file(path: Path, dictionary_list: List) -> None:
             raise CSVWriteError(f'Failed to write {path} CSV due to {str(e)}') from e
 
 
-class CSVExporter():
+class CSVExporter(CustomizableDateMixin):
 
     def __init__(
             self,
             database: 'DBHandler',
             user_directory: Path,
             create_csv: bool,
     ):
+        super().__init__(database=self.database)
```

Comment:
You called `self.database` but that attribute is later initialized. Should be `database`

Reply:
yes this is why the tests failed